src/com/company/topic2/Persoana.java

```diff
@@ -0,0 +1,34 @@
+package com.company.topic2;
+
+public class Persoana {
+    public Persoana(String numeAtribuit, String prenumeAtribuit, int virstaAtribuita, String genAtribuit, double greutateAtribuita) {
+        nume = numeAtribuit;
+        prenume = prenumeAtribuit;
+        virsta = virstaAtribuita;
+        gen = genAtribuit;
+        greutate = greutateAtribuita;
+
+    }
+
+    String nume;
+    String prenume;
+    private int virsta;
+    String gen;
+    private double greutate;
+
+
+    public void afiseazaRezultatul(double greutateAtribuita) {
+        if (greutateAtribuita > 80) {
+            System.out.println("Persoana data are nevoie de antrenamente");
+        } else {
+            this.greutate = greutateAtribuita;
+            System.out.println("Persoana are o greutate corespunzatoare");
+        }
+
+    }
+
```

Comment:
next time please create a setter for Greutate or similar attributes. I saw that you used the initialization within the Constructor method. It is good, but for practice would be good to define setters and getters.